### DISCUSSION TITLE
Welcome to mpc-autofill Discussions!

### DISCUSSION THREAD

**Discussion by chilli-axe** [chilli-axe/mpc-autofill#129] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

**Comment by MKaiserW** [comment]
Im looking at issue 173. When is the process to contribute to the codebase? Where should discussion about how to fix the issue take place?

**Reply by ndepaola** [reply]
howdy! thanks for stopping by.

this is largely a solo project so we don't have any rigid processes for this stuff. i'm happy for discussion regarding the code design for the solution to happen in the issue thread - or feel free to open a new thread in Discussions if you feel that's more appropriate!

in terms of the process of contributing - just the usual process of forking the repo, making your changes, and submitting a PR explaining the changes you've made. we have a PR template to help with this part too.

**Reply by MKaiserW** [reply]
I put something on the issue its self. Let me know if you have any thoughts otherwise ill just try number 6 and see if it works out.

**Comment by Will96Reis** [comment]
Good afternoon!
The MPCFill front end site it's not working?

**Reply by ndepaola** [reply]
howdy - i don't have direct control over any public deployments of this project's backend server. for mpcfill.com, you're looking for https://github.com/Investigamer/mpc-autofill

**Comment by uvsq22101699** [comment]
hey

I have an issue with the MacOS Version, the next it has works really well so I don't know why it don't now.
when I use the terminal, after the command :  "% chmod  +x autofill-macos" the terminal respond me with : " chmod: Unable to change file mode on autofill-macos: Operation not permitted".
Do you know how to fix this problem ?

thanx for the person who could help me (and sorry for my English)

**Reply by ndepaola** [reply]
howdy - see this issue for details https://github.com/chilli-axe/mpc-autofill/issues/211

**Comment by bjarkehvasskure** [comment]
Hi, im having a problem when running the executable - it says "zsh: bad CPU type in executable" and does not start the process? Am I making a mistake somewhere or is this an issue with software? thanks for all help!

**Reply by ndepaola** [reply]
hi there - can i confirm if you're running macOS?

there are two macOS releases, and the one you should use depends on when your computer was built. if it's a 2020 or newer model, use the ARM release; if it's 2019 or older, use the Intel release. let me know how you go with this!

**Reply by bjarkehvasskure** [reply]
ah this was the problem - im running macOS on old computer and i was using ARM! this works!! thank you!

**Comment by MKaiserW** [comment]
Maybe I'm missing something but while the mpcfill website was down I thought I would try getting the backend up myself but the drives list is empty. Are those populated with something else, or are those secret since it gives access to the drives write wise?

**Reply by ndepaola** [reply]
howdy - this repo only comes with one drive pre-configured (which is mainly for testing purposes): https://github.com/chilli-axe/mpc-autofill/blob/master/MPCAutofill/drives.example.csv

to use this, you create a copy of this file called `drives.csv` and fill it out, then run `python manage.py import_sources` to bring them into the database

afaik the drive list for mpcfill.com is not public

**Reply by MKaiserW** [reply]
Had a feeling but I thought I would ask. If I find anything out ill report back

**Comment by calliopekitty** [comment]
no XML files found in this directory issue

i tried putting it on my desktop as i saw in another thread somewhere. didnt work. it was also in my downloads folder before and that didnt work either. latest version. what do i do

**Reply by ndepaola** [reply]
howdy - sounds like you may be having this issue https://github.com/chilli-axe/mpc-autofill/discussions/214

**Comment by sl868767** [comment]
Hi, I'm having an issue on the Windows version. When it asks  (Do you want the tool to continue editing an existing project? (Press Enter if you're not sure.) And i respond, it gives me this error message (Message: invalid session id: session deleted as the browser has closed the connection
from disconnected: not connected to DevTools.)
what should i do?

**Comment by VENOM1387** [comment]
Hello, I have custom card art for Unique Cards I would like to upload to the MPC Autofill Database. I have not been able to find out how to do this yet. Any help would be appreciated.
